pkg/server/router/did.go

```diff
@@ -128,6 +133,56 @@ func (dr DIDRouter) CreateDIDByMethod(ctx context.Context, w http.ResponseWriter
 	return framework.Respond(ctx, w, resp, http.StatusCreated)
 }
 
+// toCreateDIDRequest converts CreateDIDByMethodRequest to did.CreateDIDRequest, parsing options according to method
+func toCreateDIDRequest(m didsdk.Method, request CreateDIDByMethodRequest) (*did.CreateDIDRequest, error) {
```

Comment:
help me understand...the option object can accept anything - for ion need to accept an array of public keys or service endpoints

the object is a generic `any` but to extract the value you need to know the concrete type

if it can be simplified let's do it I couldn't figure out a simpler way

Reply:
oh yeah, this can work. the tradeoff is then having an "any" request vs a more generic method like I have and specific impls per type...with then the tradeoff of having some ugly mapping code.

I think the real solution is to get away from the generic -> specific funnel code we have now and instead register first-party handlers...I'm not entirely sure how this would work it would be a significant refactor but could be a more natural approach than forcing all DID methods to follow a similar structure.

Reply:
I think it's related to Andres's points here https://github.com/TBD54566975/ssi-service/issues/362 so I'll add a note with your comment and we can address in there